packages/next/src/middleware/sitemaps/createSitemaps.ts

```diff
@@ -213,6 +227,20 @@ export async function handleSitemapPath(
     unpairedTags: ['xml', 'xml-stylesheet'],
     processEntities: true,
     htmlEntities: true,
+    /**
```

Comment:
There is code duplication here. is there a way to share config between those two functions?